notify.go

```diff
@@ -0,0 +1,69 @@
+// +build linux
+
+package seccomp
+
+import "os"
+
+// #cgo pkg-config: libseccomp
+// #include <stdlib.h>
+// #include <seccomp.h>
+import "C"
+
+// ScmpNotifReq is a seccomp notifier request
+type ScmpNotifReq struct {
+	inner C.seccomp_notif
+}
+
+// ScmpNotifResp is a seccomp notifier response
+type ScmpNotifResp struct {
+	inner C.seccomp_notif_resp
+}
+
+// NotifyAlloc creates a new pair of notification request/response structures.
+// See libseccomp API `seccomp_notify_alloc()`.
+func NotifyAlloc() (*ScmpNotifReq, *ScmpNotifResp, error) {
+	req := &ScmpNotifReq{}
+	resp := &ScmpNotifResp{}
+	if ret := C.seccomp_notify_alloc(req.inner, resp.inner); ret != 0 {
+		return nil, nil, errRc(ret)
+	}
+	return req, resp, nil
+}
+
+// NotifyFree will cleanup a pair of notification request/response structures.
+// See libseccomp API `seccomp_notify_free()`.
+func NotifyFree(req *ScmpNotifReq, resp *ScmpNotifResp) {
+	C.seccomp_notify_free(req.inner, resp.inner)
+}
+
+// Receive tries to get a notification from a seccomp notification file.
+// See libseccomp API `seccomp_notify_receive()`.
+func (s *ScmpNotifReq) Receive(file *os.File) error {
+	if ret := C.seccomp_notify_receive(file.Fd(), s.inner); ret != 0 {
+		return errRc(ret)
+	}
+	return nil
+}
+
+// Repond tries to send a notification to a seccomp notification file.
```

Comment:
`make lint` is failing because of a typo here

`s/Repond/Respond`

Reply:
Thanks for the hint! :pray: Fixed that issue.